### PR TITLE
CHANGELOG: release 0.104.0

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Mainnet confirmed up to the current height:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ go run scripts/compare-states/compare-states.go http://localhost:10332 http://localhost:10232
at 0: 58a5157b7e99eeabf631291f1747ec8eb12ab89461cda888492b17a301de81e8 vs 58a5157b7e99eeabf631291f1747ec8eb12ab89461cda888492b17a301de81e8
at 4483627: f575182fc403202a706abb5e398df0e6643b01915a4279140c91ca057d0f5894 vs f575182fc403202a706abb5e398df0e6643b01915a4279140c91ca057d0f5894
```

Testnet confirmed:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ go run scripts/compare-states/compare-states.go http://localhost:20332 http://seed1t5.neo.org:20332
at 0: 62fd8ff9b0543aea352257db5b00bbb01d1bc0d2cc665e1f24cf5de0d16ebc7b vs 62fd8ff9b0543aea352257db5b00bbb01d1bc0d2cc665e1f24cf5de0d16ebc7b
at 3078762: 7e68ba8adac968c774459adb2c2d0ee3af5d11f26bb19f38f3a5e53e435c9f0e vs 7e68ba8adac968c774459adb2c2d0ee3af5d11f26bb19f38f3a5e53e435c9f0e
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,66 @@
 
 This document outlines major changes between releases.
 
+## 0.104.0 "G" (27 Nov 2023)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,66 @@
...
+block TODO and T5 testnet up to block TODO). Please, ensure your node configuration
+doesn't contain `NativeActivations` protocol configuration section as this logic
+is hidden under Hardforks starting from the current release.
+
+New features:
+* System.Runtime.CurrentSigners interop allowing to get signers of the currently
+  loaded transaction (#3058)
+* add `strLen` method to native StdLib contract (#3208)
+* introduce transaction attributes pricing regulation via native PolicyContract (#3155)
+* `canceltx` CLI command as a replacement to unsupported `canceltransaction` RPC
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: How about an "alternative"? I'd like to mention here that we won't support `canceltx` RPC.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,66 @@
 
 This document outlines major changes between releases.
 
+## 0.104.0 "G" (27 Nov 2023)
+
+We're updating NeoGo to push out a number of useful updates and protocol
+extensions as well as make it compatible with 3.6.2 version of C# node. The most
+vital behaviour changes are VM-level protocol constraints imposed on the size of
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,66 @@
...
+serialized stackitems and `NativeActivation` node setting removal. This version
+also includes a set of small but meaningful changes in the interoperability layer
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,66 @@
...
+* introduce serialization limit to stackitem.Item and fail large contracts deploy
+  wrt this setting (#3218)
+* add customizable `MaxRequestBodyBytes` and `MaxRequestHeaderBytes` RPC server
+  configuration setting (#3221)
+
+Improvements:
+* provide more detailed error on attempt to read non-existent service wallet
+  provided via node configuration file (#3210)
+* optimize emit of imported code for autogenerated RPC bindings (#3215)
+* improve documentation for storage Seek-related methods (#3222)
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

**Comment by AnnaShaleva** [comment]
Rebased onto current master.

**Comment by AnnaShaleva** [comment]
Bench is unhealthy.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**